docs/library/urandom.rst

```diff
@@ -0,0 +1,49 @@
+:mode:`urandom` -- generates random numbers
+===========================================
+
+.. module:: urandom
+   :synopsis: random numbers
+
+This module implements pseudo-random number generator. The functions that
+these module provides are bond methods of a hidden number of :random.Random:
+class.   
+
+Given below are the methods that this module provides.
+
+  .. method:: urandom 
+
+    Returns an n-bit pseudo-random number(output is determinsitic).
```

Comment:
Missing space. Spelling.

```suggestion
    Returns an n-bit pseudo-random number (output is deterministic).
```

Reply:
And from a higher-level perspective, the pseudo-randomness/determinism applies to the entire module, not just this method. So it would make sense to move that part above to the module description.